Core/include/Acts/Utilities/IAxis.hpp

```diff
@@ -58,6 +62,31 @@ class IAxis {
   /// @return total number of bins (excluding under-/overflow bins)
   virtual std::size_t getNBins() const = 0;
 
+  /// Centralized axis factory for equidistant binning
+  ///
+  /// @param aBoundaryType the axis boundary type
+  /// @param min the minimum edge of the axis
+  /// @param max the maximum edge of the axis
+  /// @param nbins the number of bins
+  ///
+  /// @throws std::invalid_argument if min >= max or nbins == 0
+  ///
+  /// @return a unique pointer to the axis
+  static std::unique_ptr<IAxis> create(AxisBoundaryType aBoundaryType,
```

Comment:
We could consider making these explicit with `createEquidistant` and `createVariable`